### PR TITLE
docs(aio): make glossary links work again

### LINKED ISSUE #16106: The Alphabet index on the glossary content page is broken and kind of useless
**Current behavior**
The glossary page contains links to various letters of the alphabet like: https://aio-staging.firebaseapp.com/guide/glossary#J

------
screen shot 2017-04-18 at 7 11 50 pm

------

Unfortunately most links don't do anything.

I'd actually argue that this kind of table of content is not that useful, we should instead add a new section into the search results that would deep link to the glossary.

### PR DESCRIPTION
**Pull request by Foxandxss**
closes #16201

### CODE REVIEW

**Comment by gkalpak** [comment]
This only partly addresses #16106 :smiley:

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -727,7 +728,7 @@ Observables are a proposed feature for ES2016, the next version of JavaScript.
 
 </div>
 
-
+{@a O}
```
Comment (gkalpak) [inline aio/content/guide/glossary.md]: Wrong place.

**Review by gkalpak — DISMISSED** [review]

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (Foxandxss) [inline aio/content/guide/glossary.md]: Why?

**Comment by Foxandxss** [comment]
That is what happens when you read only the title on the project table.

**Review by petebacondarwin — APPROVED** [review]
LGTM - despite @gkalpak's confusing request.

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (gkalpak) [inline aio/content/guide/glossary.md]: Because there is `Observable` (which also starts with `O`) right above.

**Comment by gkalpak** [comment]
George says to Ward and Pete "It's not OK" (in his not-humble opinion).

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (Foxandxss) [inline aio/content/guide/glossary.md]: Good `Observation` George.

**Comment by Foxandxss** [comment]
I fixed the error.

**Review by petebacondarwin — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

**Pull request merged**